__tests__/app.test.js

```diff
@@ -118,6 +118,23 @@ describe('/api/articles', () => {
                     })
                 })
         })
+        describe('topic query', () => {
+            it('uses a provided topic query to filter output', () => {
+                return request(app).get('/api/articles').query({ topic: 'cats' })
+                    .expect(200)
+                    .then(({ body: { articles }}) => {
+                        expect(articles).toHaveLength(1)
```

Comment:
I would test that the topic property of the article objects is the topic that you expect